designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/scenarioActivity/Dtos.scala

```diff
@@ -107,105 +116,105 @@ object Dtos {
     private val commentRelatedActions = List("delete_comment", "edit_comment")
 
     case object ScenarioCreated extends ScenarioActivityType {
-      override def displayableName: String        = "Scenario created"
-      override def icon: String                   = "/assets/activities/scenarioModified.svg"
-      override def supportedActions: List[String] = List.empty
+      override def displayableName(scenarioType: ScenarioType): String = s"${scenarioType.entryName} created"
```

Comment:
It looks very weird to me. 
We have an activity type that produces a displayable name? And it needs scenatioType to do that?
IMO the sth that produces a displayable name needs a Scenario activity (and maybe its type), but not the other way around.

Is it a way to improve this part of code?

Reply:
Refactored a bit:
- separate displayableNameForScenario and displayableNameForFragment are explicitly defined
- the correct one is chose, when the service creates metadata for scenario/fragment